include/seqan3/index/detail/csa_alphabet_strategy.hpp

```diff
@@ -0,0 +1,128 @@
+#pragma once
```

Comment:
do we need this stuff in SeqAn? Can you not add it to the SDSL? If indeed it has to be in our library then it need to follow our code guideline and contain a copyright notice....